tests/pipeline/test_quarters_estimates.py

```diff
@@ -888,6 +900,12 @@ def create_expected_df(cls, tuples, end_date):
         df.index = df.index.rename('knowledge_date')
         df['at_date'] = end_date.tz_localize('utc')
         df = df.set_index(['at_date', df.index.tz_localize('utc')]).ffill()
+        new_sids = set(cls.get_sids()) - set(df.columns)
+        df = pd.concat([df, pd.DataFrame(columns=new_sids)])
+        # Have to explicitly cast to float so that nans are of type
+        # np.float64 - otherwise they won't compare equal.
+        for col in new_sids:
```

Comment:
Still unsure of what you mean. Do you mean why we need sids in `assets` that aren't used in the raw data?